### PR TITLE
Cu yf64cx  navigation desktop

### PR DESCRIPTION
**Pull request by the0ffh**
## Checklist

- [ ] Added or extended tests
- [ ] Checked linter warnings
- [ ] Typed as much as possible

### CODE REVIEW

**Comment by the0ffh** [comment]
Task linked: [CU-yf64cx Navigation Desktop](https://app.clickup.com/t/yf64cx)

File: core/src/components/Navigation/NavigationItem.tsx [hunk 1]
```diff
@@ -42,8 +42,8 @@ const NavigationItem: React.FunctionComponent<NavigationItemProps> = ({
       }}
     >
       {iconName && (
-        <ListItemIcon aria-label={label}>
-          <Icon name={iconName} />
+        <ListItemIcon aria-label={label} style={{ minWidth: 0 }}>
```
Comment (LianeWalter) [inline core/src/components/Navigation/NavigationItem.tsx]: why is this hardcoded like this?

**Review by LianeWalter — APPROVED** [review]
the best PR that ever was and ever will be.

**Comment by LianeWalter** [comment]
🥇

**Pull request merged**